CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.4.0
```

Comment:
I would be just fine with doing 1.0 now.  I don't see any reason we need to avoid semver at this point.  In fact it's probably overdue since we have shipped this, it's running in production, multiple teams using it, etc. etc.